Desktop/Automation Home Tasks/HomeTask5/src/test/java/com/epam/automation/testsTestNG/DivCalcTest.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.automation.testsTestNG;
+
+import com.epam.automation.BaseConfigurations.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class DivCalcTest extends BaseTest {
+
+    @Test(groups = "TestDivision", dataProvider = "DoubleValuesForDivision")
+    public void testDivOperationWithDoubleValues(double a, double b, double expected){
+       double result =  calculator.div(a, b);
+       Assert.assertEquals(result,expected, "Incorrect result of division operation with double params");
+    }
+
+
+    @DataProvider(name = "DoubleValuesForDivision")
+    Object [][] DoubleValuesForDivision(){
+        return new Object[][]{
+                {1.0, 1.0, 1.0},
+                {45.5, 1.0, 45.5},
+                {90.0, 10.0, 9.0 },
+                {135.0, -1.0, -135.0},
+                {0, 120.0, 0}
+        };
+    }
+
+    @Test(groups = "TestDivision", dataProvider = "LongValuesForDivision")
+    public void testDivOperationWithLongValues(long a, long b, long expected){
+        long result =  calculator.div(a,b);
+        Assert.assertEquals(result, expected, "Incorrect result of division operation with long params");
+    }
+
+
+    @DataProvider(name = "LongValuesForDivision")
+    Object [][] LongValuesForDivision(){
+        return new Object[][]{
+                {1, 1, 1},
```

Comment:
use long, not int values

Reply:
As for manual testing, you should use demonstrative values, which could discover defects. For instance, 
how  {45, 1, 45}, {90, 10, 9 }, cases use something different? 1 of them is redundant.
Border values should be used. for example -1, 0,1, max long, min long, etc.